vscode-dotnet-runtime-library/src/Utils/ErrorHandler.ts

```diff
@@ -126,17 +125,15 @@ export async function callWithErrorHandling<T>(callback: () => T, context: IIssu
 
 async function configureManualInstall(context: IIssueContext, requestingExtensionId: string): Promise<void> {
     const manualPath = await context.displayWorker.displayPathConfigPopUp();
-    if (manualPath && fs.existsSync(manualPath))
-    {
-        try
-        {
-            let configVal: IExistingPath[] = [{ [ExistingPathKeys.extensionIdKey]: requestingExtensionId, [ExistingPathKeys.pathKey] : manualPath}];
+
+    if (manualPath && fs.existsSync(manualPath)) {

```

Comment:
```suggestion
    if (manualPath && fs.existsSync(manualPath))
    {
```

nit: preserve the style choices of the original document (same for the try catch block)